src/files-breadcrumbs.tsx

```diff
@@ -105,12 +105,12 @@ export function FilesBreadcrumbs({ path, showHidden, setShowHidden }: { path: st
                     />}
                 {!editMode && fullPath.map((dir, i) => {
                     return (
-                        <React.Fragment key={dir || "/"}>
+                        <React.Fragment key={fullPath.slice(0, i).join("/") || "/"}>
                             <Button
                               isDisabled={i === path.length - 1}
                               icon={i === 0 ? <HddIcon /> : null}
                               variant="link" onClick={() => { navigate(i + 1) }}
-                              key={dir} className="breadcrumb-button"
+                              key={`${dir}-${i}`} className="breadcrumb-button"
```

Comment:
This still uses `i`. But why does hthis need a key at all? It's not a dynamic list inside the fragment.

Reply:
Right, dropped it.